examples/fastai/train.py

```diff
@@ -0,0 +1,42 @@
+#
+# This short example is based on the fastai GitHub Repository of vision examples
+# https://github.com/fastai/fastai/blob/master/examples/vision.ipynb
+# Modified here to show mlflow.fastai.autolog() capabilities
+#
+import argparse
+from fastai.vision import *
+import mlflow.fastai
+
+def parse_args():
+   parser = argparse.ArgumentParser(description='Fastai example')
```

Comment:
Indentation should be 4 spaces.

Reply:
Will have to set my settings in PyCharm to be 4 spaces :-)

Reply:
I feel like we should apply lint checks to the examples too (we currently don't).